start/src/main/java/org/apache/accumulo/start/classloader/vfs/ContextManager.java

```diff
@@ -158,14 +158,6 @@ public synchronized void setContextConfig(ContextsConfig config) {
     this.config = config;
   }
 
-  public boolean isKnownContext(String contextName) {
```

Comment:
The `public` modifier was added to the class in #3678 , that could be backed out as well.

Reply:
I removed public from the class